podvm/qcow2/rhel/qemu-rhel.pkr.hcl

```diff
@@ -14,12 +14,13 @@ source "qemu" "rhel" {
   iso_checksum     = "${var.cloud_image_checksum}"
   iso_url          = "${var.cloud_image_url}"
   output_directory = "output"
-  qemuargs         = [["-m", "${var.memory}"], ["-smp", "cpus=${var.cpus}"], ["-cdrom", "${var.cloud_init_image}"], ["-serial", "mon:stdio"], ["-cpu", "Cascadelake-Server"]]
```

Comment:
Did you have a chance to test with x86? i recall we needed to specify cpu type for CentOS otherwise it failed, see 90ccfc3fa0ee56c2fc61b23ecf3731f819faf875
cc @bpradipt as he wrote this line